client/k8s/k8s.go

```diff
@@ -44,7 +43,6 @@ func newRawClientConfig(config Config) *rest.Config {
 		Username:        config.Username,
 		Password:        config.Password,
 		BearerToken:     config.BearerToken,
-		Insecure:        config.Insecure,
```

Comment:
100% there is also a TODO comment in the code to do it but this here is just vendor deps and it was painful enough for the moment. :D